Source/Scene/Model.js

```diff
@@ -556,6 +558,17 @@ define([
         // CESIUM_RTC extension
         this._rtcCenter = undefined;    // in world coordinates
         this._rtcCenterEye = undefined; // in eye coordinates
+
+        // TODO : make a material property?
```

Comment:
Why are these private?  Why not just `DOC_TBA`?  The user may want to override these, e.g., only have the 3D model that the camera is following cast a shadow for performance.